src/lib/components/AddItem/AddItem.styled.ts

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+import { Button } from '../commons/Button';
+
+export const StyledAddItemButton = styled(Button)`
+  height: 1rem;
```

Comment:
need to make some prop size='S'|'M'|'L' so we can make the buttons behave similarlly